### PR TITLE
refactor scatter vs violin plot

### PR DESCRIPTION
**Pull request by CloseChoice**
## Overview

Supports #3036  

Description of the changes proposed in this pull request:
- fixed typos and improved descriptions.

## Checklist

- [x] All [pre-commit checks](https://pre-commit.com/#install) pass.
- ~[ ] Unit tests added (if fixing a bug or adding a new feature)~

### CODE REVIEW

File: notebooks/tabular_examples/tree_based_models/Scatter Density vs. Violin Plot Comparison.ipynb [hunk 1]
```diff
@@ -173,7 +181,7 @@
    "source": [
     "## Violin plot\n",
     "\n",
-    "These are a standard violin plot but with outliers drawn as points. This gives a more accurate representation of the density out the outliers than a kernel density estimated from so few points. The color represents the average feature value at that position, so red regions have mostly high valued feature values while blue regions have mostly low feature values."
+    "These are a standard violin plot but with outliers drawn as points. This gives a more accurate representation of the density out the outliers than a kernel density estimated from only a few points. The color represents the average feature value at that position, so n red regions high feature values dominate while blue regions low feature values dominate."
```
Comment (connortann) [inline notebooks/tabular_examples/tree_based_models/Scatter Density vs. Violin Plot Comparison.ipynb]:
```suggestion
    "These are a standard violin plot but with outliers drawn as points. This gives a more accurate representation of the density out the outliers than a kernel density estimated from only a few points. The color represents the average feature value at that position, so red regions high feature values dominate while blue regions low feature values dominate."
```

File: notebooks/tabular_examples/tree_based_models/Scatter Density vs. Violin Plot Comparison.ipynb [hunk 2]
```diff
@@ -125,7 +133,7 @@
    "source": [
     "## Dot plot\n",
     "\n",
-    "This combines a scatter plot with density estimation by letting dots pile up when they don't fit. The advatange of this approach is that it does not hide anything behind kernel smoothing, so what-you-see-is-what-is-there."
+    "The dot plot combines a scatter plot with density estimation by letting dots pile up when they don't fit. The advatange of this approach is that it does not hide anything behind kernel smoothing, so what-you-see-is-what-is-there."
```
Comment (connortann) [inline notebooks/tabular_examples/tree_based_models/Scatter Density vs. Violin Plot Comparison.ipynb]:
```suggestion
    "The dot plot combines a scatter plot with density estimation by letting dots pile up when they don't fit. The advantage of this approach is that it does not hide anything behind kernel smoothing, so you see an exact representation of the data."
```

File: notebooks/tabular_examples/tree_based_models/Scatter Density vs. Violin Plot Comparison.ipynb [hunk 3]
```diff
@@ -6,14 +6,22 @@
...
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "[22:05:34] WARNING: C:\\buildkite-agent\\builds\\buildkite-windows-cpu-autoscaling-group-i-07f6e447eee219473-1\\xgboost\\xgboost-ci-windows\\src\\c_api\\c_api.cc:1240: Saving into deprecated binary model format, please consider using `json` or `ubj`. Model format will default to JSON in XGBoost 2.2 if not specified.\n"
+     ]
+    }
+   ],
```
Comment (connortann) [inline notebooks/tabular_examples/tree_based_models/Scatter Density vs. Violin Plot Comparison.ipynb]:
```suggestion
   "outputs": [],
```

**Review by connortann — CHANGES_REQUESTED** [review]
Looks good, a few suggestions above

**Comment by connortann** [comment]
I edited the PR description slightly as this will help #3036 but not fully close the issue.

**Comment by CloseChoice** [comment]
Thanks @connortann for the review. I incorporated all your suggestions.

**Review by connortann — APPROVED** [review]
LTGM! Nice one.

**Pull request merged**